app/components/Settings/OptionsDropdown/OptionsDropdown.react.js

```diff
@@ -1,61 +1,46 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
-import {flatten, keys} from 'ramda';
-import Select from 'react-select';
+import SQLOptions from './SQLOptions.react';
+import ESIndicesOptions from './ESIndicesOptions.react';
+import ESDocsOptions from './ESDocsOptions.react';
 
 export default class OptionsDropdown extends Component {
-    constructor(props) {
-        super(props);
-        this.renderSQLOptions = this.renderSQLOptions.bind(this);
-        this.renderElasticsearchIndecies = this.renderElasticsearchIndecies.bind(this);
-        this.renderElasticsearchDocs = this.renderElasticsearchDocs.bind(this);
-    }
 
     static propTypes = {
         // See type of react-select's Select `value` property
```

Comment:
I expected `selectedTable` and  selectedIndex` to be `string` (since we used as keys).
Would `PropTypes.string` break anything?